dcicutils/s3_utils.py

```diff
@@ -187,7 +187,12 @@ def read_s3_zipfile(self, s3key, files_to_extract):
                 ret_files[name] = zipstream.open(zipped_filename).read()
         return ret_files
 
-    def unzip_s3_to_s3(self, zipped_s3key, dest_dir, acl=None):
+    def unzip_s3_to_s3(self, zipped_s3key, dest_dir, acl=None, do_not_store=False):
```

Comment:
Why isn't this `store_result=True`? It hurts my head to be looking at double negatives.

Reply:
Yeah I know, either way works but since the default is store and we have used this function for years assuming it will store the data, I wanted to make it explicit that we do not want to store it.

Reply:
I think a default of `store=True` still makes it explicit in that
* unzip_s3_to_s3(zipped_key, dir, acl=acl) is the same as always, just implying store=True
* unzip_s3_to_s3(zipped_key, dir, acl=acl, store=False) is explicit about not storing.

Although really a name that said what is not being stored might be helpful, regardless of what the boolean sense of the flag is.

It sounds weird to say, but It feels like if this argument is given you're saying "take the time to do the operation, but don't put the output anywhere". Other than just consuming time, I don't see what is going on in that case. It might be doing something and I might not be seeing it, so please do feel free to point to what it doees other than that one 'put' that is either being suppressed or not. Surely the argument is not only about suppressing the return value, since it doesn't say "do_not_tell_me_what_you_did" but instead says "do_not_do_something".

Sorry if I seem to be repeating myself 6 different ways, but I'm trying to offer you different ways to see my confusion and fix me. :)